app.js

```diff
@@ -3,9 +3,18 @@ const app = express()
 
 const {getTopics} = require('./controllers/topics.controllers')
 const {getEndpoints} = require('./controllers/api.controllers')
+const {getArticleById} = require('./controllers/articles.controllers')
 
 app.get('/api/topics', getTopics)
 
 app.get('/api', getEndpoints)
 
+app.get('/api/articles/:article_id', getArticleById)
```

Comment:
Could have a psql error handler for psql errors you get. see the comment on your model